samples/msal2provider-asp-net-core-sso/Pages/Shared/_Layout.cshtml

```diff
@@ -12,7 +12,7 @@
 
   @if (User.Identity.IsAuthenticated)
   {
-<script src="https://unpkg.com/@@microsoft/mgt/dist/bundle/mgt-loader.js"></script>
+<script src="https://unpkg.com/@@microsoft/mgt@@2/dist/bundle/mgt-loader.js"></script>
```

Comment:
Typo but can't suggest the change for whathever reason.

Reply:
Strangely when I tested this code it worked as expected....